integration-tests/smoke/keeper_test.go

```diff
@@ -507,6 +511,50 @@ func getKeeperSuite(
 					}, "30s", "1s").Should(Succeed())
 				})
 			}
+
+			if testToRun == RemovingKeeperTest {
+				It("removes one keeper and makes sure the upkeeps are not affected by this and still perform", func() {
+					var initialCounters = make([]*big.Int, len(upkeepIDs))
+					// Make sure the upkeeps are running before we remove a keeper
+					Eventually(func(g Gomega) {
+						for upkeepID := 0; upkeepID < len(upkeepIDs); upkeepID++ {
+							counter, err := consumers[upkeepID].Counter(context.Background())
+							initialCounters[upkeepID] = counter
+							g.Expect(err).ShouldNot(HaveOccurred(), "Failed to get counter for upkeep "+strconv.Itoa(upkeepID))
+							g.Expect(counter.Cmp(big.NewInt(0)) == 1, "Expected consumer counter to be greater than 0, but got %s", counter)
```

Comment:
can you check > 0 here, instead of == 1. It might be possible we directly see 2 if upkeep gets performed multiple times